.github/workflows/go.yaml

```diff
@@ -0,0 +1,77 @@
+name: go
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+  pull_request:
+
+# TODO(bwplotka): Add tests here.
```

Comment:
e2e tests or unit tests? The first one has been done.

Reply:
Unit tests (: 

They are still in the Circle CI for now.